documentation/puppetdb_connection.markdown

```diff
@@ -64,6 +64,21 @@ This setting can let the Puppet master stay partially available during a PuppetD
 
 The default value is false.
 
+## Differences in Puppet Enterprise
+
+### `include_unchanged_resources`
```

Comment:
Another way to do this, have the setting on its own, but say (PE Only) in the title. Just thinking out loud.